app/electron/init.html

```diff
@@ -212,6 +212,10 @@ <h2>思源笔记</h2>
             <option value="zh_CHT">繁體中文</option>
             <option value="ja_JP">日本語</option>
             <option value="it_IT">Italiano</option>
+			<option value="de_DE">Deutsch</option>
```

Comment:
problem with code indentation

Reply:
Oh, thought I'd fixed that already apologies.